src/open_clip/hf_model.py

```diff
@@ -91,3 +137,9 @@ def lock(self, unlocked_layers:int=0, freeze_layer_norm:bool=True):
         for n, p in self.transformer.named_parameters():
             if True: #mb optional LayerNorm params etc.
                 p.requires_grad = False
+
+# temporary for fast testing
```

Comment:
keeping this in?

Reply:
guess we might have different development routines, so let's keep it separate, removed